applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -587,19 +586,7 @@ static void stream_recv_cb(struct bt_bap_stream *stream, const struct bt_iso_rec
 		return;
 	}
 
-	uint32_t octets_per_frame = bt_codec_cfg_get_octets_per_frame(stream->codec);
-
-	if (buf->len != octets_per_frame && bad_frame != true) {
-		data_size_mismatch_cnt++;
-		if ((data_size_mismatch_cnt % 500) == 1) {
-			LOG_DBG("Data size mismatch, received: %d, codec: %d, total: %d", buf->len,
-				octets_per_frame, data_size_mismatch_cnt);
-		}
-
-		return;
-	}
-
-	receive_cb(buf->data, buf->len, bad_frame, info->ts, channel_index);
+	receive_cb(buf->data, buf->len, bad_frame, info->ts, channel_index, bt_codec_cfg_get_octets_per_frame(stream->codec));
```

Comment:
bt_codec_cfg_get_octets_per_frame() brings delay in BIS headset can cause audio_datapath failed, but not sure if it is also an issue for CIS, or in latest CIS code, maybe this won't be a blocker for the current code.

Reply:
I chose not to alter that in this code.